dev-requirements.txt

```diff
@@ -1,18 +1,14 @@
 pytest
-doctr
+pytest-cov
 ipython
 matplotlib >=3.2.0
 ophyd >=1.5.0
 pymongo
 mongomock >=3.22.0
-sphinx
-sphinx_rtd_theme
 # Removed temporarily, pip installations fail lacking gssapi libs
 # psdm_qs_cli
-sphinx-click
 # PyPI's 2020 resolver can be problematic with these
 # dependencies-of-dependencies.  Constrain them here:
 six >=1.10.0
```

Comment:
I wonder if we still need this

Reply:
Now's as good a time as any to check 👀

Reply:
Seems like it's no longer necessary 👍   At least on CI